data-scripts/polisis.js

```diff
@@ -0,0 +1,29 @@
+const fs = require('fs')
+const polisisData = require('../data/symlinked/polisis')
+const reasonsWithScore = {
+    "Certain data is shared with third parties for advertising purposes.": 4,
+    "Some personal information is shared with third parties.": 6,
+    "Data might be shared in the case of a merger or acquisition.": 1,
+    "Some data might be retained indefinitely.": 1
+}
+
+let outputData = {}
+
+Object.keys(polisisData).forEach((url) => {
```

Comment:
It seems like polisisData might change more than once in the future - I know we're reading from file, but maybe it's worth checking if it's an object and failing gracefully otherwise before attempting iteration on the keys.